imap_processing/hi/l1b/hi_l1b.py

```diff
@@ -26,12 +33,16 @@ def hi_l1b(l1a_cdf_path: Path):
     processed_data : xarray.Dataset
         Processed xarray dataset
     """
-    l1a_dataset = load_cdf(l1a_cdf_path)
-    packet_enum = HIAPID(l1a_dataset["pkt_apid"].data[0])
+    logger.info(f"Running Hi L1B processing on file: {l1a_cdf_path.name}")
+    l1a_dataset = load_cdf(l1a_cdf_path, to_datetime=True)
```

Comment:
After your last PR I think this can be removed now.
```suggestion
    l1a_dataset = load_cdf(l1a_cdf_path)
```

Reply:
Good catch!